docs/doxygen/config.doxyfile

```diff
@@ -269,7 +269,11 @@ TAB_SIZE               = 4
 # commands \{ and \} for these it is advised to use the version @{ and @} or use
 # a double escape (\\{ and \\})
 
-ALIASES                =
+ALIASES                = "transportcallback=@ingroup mqtt_callback_types" \
+                         "transportstruct=@ingroup mqtt_struct_types" \
+                         "transportpage=@page mqtt_transport_interface Transport Interface" \
+                         "transportsectionimplementation=@section mqtt_transport_interface_implementation Implementing the Transport Interface" \
+                         "transportsectionoverview=@section mqtt_transport_interface_overview Transport Interface Overview"
```

Comment:
What are these changes? They seem unrelated to updating the version of doxygen being used.

Reply:
Since the file was updated automatically, they were copied over from the bottom of the previous file. These are not required. I shall delete them.

Reply:
Sounds good. I'll approve after the tests are updated.